url-detector/src/test/java/com/linkedin/urls/TestUrl.java

```diff
@@ -117,7 +119,8 @@ public void testScheme(String testInput, String scheme, String host, String path
         {"fbeoo:@boop.com/dhdeh/@1234?aj=r", "boop.com", "http://fbeoo@boop.com/dhdeh/@1234?aj=r"},
         {"bloop:@noooo.com/doop/@1234", "noooo.com", "http://bloop@noooo.com/doop/@1234"},
         {"bah.com/lala/@1234/@dfd@df?@dsf#ono", "bah.com", "http://bah.com/lala/@1234/@dfd@df?@dsf#ono"},
-        {"https://dewd:dood@www.google.com:20/?why=is&this=test#?@Sdsf", "www.google.com", "https://dewd:dood@www.google.com:20/?why=is&this=test#?@Sdsf"}
+        {"https://dewd:dood@www.google.com:20/?why=is&this=test#?@Sdsf", "www.google.com", "https://dewd:dood@www.google.com:20/?why=is&this=test#?@Sdsf"},
+        {"http://013.xxx/","013.xxx", "http://013.xxx/"}
```

Comment:
What will happen in case of `http://0x.com/` ?

Reply:
The library will treat it as host name, not ip